segtypes/common/codesubsegment.py

```diff
@@ -273,6 +273,32 @@ def determine_symbols(self):
                 while len(possible_jtbl_jumps) > 0 and possible_jtbl_jumps[0][0] < hi_insn.address:
                     del possible_jtbl_jumps[0]
 
+                # Find gp relative reads and writes e.g  lw $a1, 0x670($gp)
+                if hi_insn.op_str.endswith("($gp)"):
+                    gp_base = options.get_gp()
+                    if gp_base is None:
+                        print("gp_value not set in yaml, can't calculate %gp_rel reloc value")
```

Comment:
This should probably be an error where we exit, telling the user to set the setting. I think there's a .error method on Segment? like self.error("..")